### PR TITLE
IGVF-1504-fix-preferred-assay-titles

### PR DESCRIPTION
**Pull request by khzlin**

### CODE REVIEW

File: src/igvfd/schemas/changelogs/measurement_set.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`measurement_set.json`*
 
+### Schema version 13
+
+* Replace `histone ChIP-seq`, `Parse Split-seq`, `Saturation genome editing`, `SHARE-Seq`, and `Yeast two-hybrid` with `Histone ChIP-seq`, `Parse SPLiT-seq` `SGE`, `SHARE-seq` and `Y2H` respectively in the enum list for `preferred_assay_title`.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/measurement_set.md:5]: please change to : "Replace `histone ChIP-seq`, `Parse Split-seq`, `Saturation genome editing`, `SHARE-Seq`, and `Yeast two-hybrid` with `Histone ChIP-seq`, `Parse SPLiT-seq` `SGE`, `SHARE-seq` and `Y2H` respectively in `preferred_assay_title`.

**Review by gabdank — CHANGES_REQUESTED** [review]
please add MPRA as well as "shorten" the change log statement.
also please add a test for upgrade.

File: src/igvfd/schemas/changelogs/measurement_set.md (hunk 1 above)
Comment (khzlin) [reply src/igvfd/schemas/changelogs/measurement_set.md]: done

**Review by gabdank — APPROVED** [review]

**Review by gabdank — DISMISSED** [review]

**Review by gabdank — DISMISSED** [review]

File: src/igvfd/schemas/changelogs/measurement_set.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 ## Changelog for *`measurement_set.json`*
 
+### Schema version 13
+
+* Renamed `histone ChIP-seq`, `Parse Split-seq`, `Saturation genome editing`, `SHARE-Seq`, and `Yeast two-hybrid` to `Histone ChIP-seq`, `Parse SPLiT-seq`, `SGE`, `SHARE-seq` and `Y2H` respectively in preferred_assay_title.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/measurement_set.md:5]: does it comply with the idea presented by Jennifer yesterday?

**Review by gabdank — CHANGES_REQUESTED** [review]
see the upgrade test comment

File: src/igvfd/schemas/changelogs/measurement_set.md (hunk 2 above)
Comment (khzlin) [reply src/igvfd/schemas/changelogs/measurement_set.md:5]: we did not get to this case yesterday. Jennifer and I will talk more about this during wrangler's meeting tomorrow.

**Review by gabdank — APPROVED** [review]

**Pull request merged**